### PR TITLE
feat(query): add SQL Server alert email disabled for Terraform

### PR DESCRIPTION
**Pull request by tiagocruz20**
**Proposed Changes**
- added a query to check if alert email enabled is true for sql server resource

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Comment by rogeriopeixotocx** [comment]
We need to review category and severity

File: assets/queries/terraform/azure/sql_server_alert_email_disabled/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "55975007-f6e7-4134-83c3-298f1fe4b519",
+  "queryName": "SQL Server Alert Email Disabled",
+  "severity": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/sql_server_alert_email_disabled/metadata.json:4]:
```suggestion
  "severity": "INFO",
```

File: assets/queries/terraform/azure/sql_server_alert_email_disabled/metadata.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "category": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/sql_server_alert_email_disabled/metadata.json:5]:
```suggestion
  "category": "Best Practices",
```

File: assets/queries/terraform/azure/sql_server_alert_email_disabled/test/positive_expected_result.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "queryName": "SQL Server Alert Email Disabled",
+    "severity": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/sql_server_alert_email_disabled/test/positive_expected_result.json:4]:
```suggestion
    "severity": "INFO",
```

File: assets/queries/terraform/azure/sql_server_alert_email_disabled/test/positive_expected_result.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    "line": 1,
+    "fileName": "positive1.tf"
+  },
+  {
+    "queryName": "SQL Server Alert Email Disabled",
+    "severity": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/azure/sql_server_alert_email_disabled/test/positive_expected_result.json:10]:
```suggestion
    "severity": "INFO",
```

File: payload.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (rogeriopeixotocx) [inline payload.json:1]: Remove this file

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**